web/src/views/DandiMetaViewer.vue

```diff
@@ -42,9 +62,15 @@
                 >
                   <v-icon>mdi-pencil</v-icon>
                 </v-btn>
+                <v-btn
+                  @click="viewContents"
```

Comment:
In general if a button's purpose is to link to a URL, we should use `:to` rather than `@click`. This allows the user to interact with the link: preview the URL, open it in a new tab, copy the URL, etc.

Reply:
Good to know, I'll fix it!